src/ray/util/logging.cc

```diff
@@ -383,6 +383,9 @@ void RayLog::InstallFailureSignalHandler(const char *argv0, bool call_previous_h
   if (is_failure_signal_handler_installed_) {
     return;
   }
+  // InitializeSymbolizer cannot be called twice, and is called in
+  // other libraries like pytorchaudio. So do not call it here.
+  // absl::InitializeSymbolizer(argv0);
   absl::InitializeSymbolizer(argv0);
```

Comment:
This is still called?

Is there a way to check if `InitializeSymbolizer` is called and skip calling it again if it's already called.

Reply:
Ahh, sorry, fixed. 
> Is there a way to check if InitializeSymbolizer is called 

I could not find one, I extended the comment. Also: this is a nice-to-have feature, not required for running. I think it makes sense to avoid it since other projects may call it again after ray initializes, and we have no control over that.